generators/wp/templates/chisel-starter-theme/templates/components/tease-post.twig

```diff
@@ -2,7 +2,7 @@
 
 {% block content %}
   <h2><a href="{{post.link}}">{{post.title}}</a></h2>
-  <p>{{post.get_preview(25)}}</p>
+  <p>{{post.preview(25)}}</p>
```

Comment:
`post.preview` is not drop-in replacement for `post.get_preview`. It returns [PostPreview](https://timber.github.io/docs/reference/timber-postpreview) object. In that case I Think it should be `post.preview().length(25)`.

Reply:
Thanks, tested and it works like that. Fixed.